app/models/link_checker_api_report/create_from_batch_report.rb

```diff
@@ -4,7 +4,7 @@ def initialize(payload, reportable)
     @reportable = reportable
   end
 
-  def create
+  def create!
```

Comment:
This doesn't seem correct since we don't have a `create` method. I'm guessing it's to appease the linter? might be better to rename it `#call` ?

Reply:
Yup, good spot - updated 👍